xl/player/player.py

```diff
@@ -397,7 +397,8 @@ def is_playing(self):
             :returns: whether the player is currently playing
             :rtype: bool
         """
-        return self._engine.get_state() == 'playing'
+        return self._engine.get_state() is not None \
```

Comment:
Hm, I'm not sure where the source of the bug is, but there does seem to be problems in the interaction between those two plugins. The `self` there is the Preview Device's player object (you can check by printing `self._name`)&mdash;which should have disappeared at that point.